packages/components/Notification/src/Notification.tsx

```diff
@@ -1,10 +1,12 @@
 /** @jsx withSlots */
-import { View } from 'react-native';
 import { notification, NotificationType, NotificationProps } from './Notification.types';
+import { Pressable } from 'react-native';
```

Comment:
For now, import `Pressable` from '@fluentui-react-native/pressable`. We have a copy of Pressable we use, that we should eventually switch off of.